src/Bicep.Core/Emit/InlineDependencyVisitor.cs

```diff
@@ -106,10 +106,16 @@ public override void VisitVariableDeclarationSyntax(VariableDeclarationSyntax sy
 
         public override void VisitFunctionCallSyntax(FunctionCallSyntax syntax)
         {
-            VisitFunctionCallSyntaxInternal(syntax);
+            VisitFunctionCallSyntaxBaseInternal(syntax);
             base.VisitFunctionCallSyntax(syntax);
         }
 
+        public override void VisitInstanceFunctionCallSyntax(InstanceFunctionCallSyntax syntax)
```

Comment:
>VisitInstanceFunctionCallSyntax [](http://example.com/codeflow?start=29&length=31)

This is a common bug class. Makes me wonder if we should have a custom analyzer to flag visitors that implement one of the overrides but not both.